enhancements/console/customize-catalogs.md

```diff
@@ -0,0 +1,179 @@
+---
+title: customize-catalogs
+authors:
+  - "@lokanandaprabhu"
+reviewers:
+  - "@jerolimov"
+  - "@spadgett"
+approvers:
+  - "@spadgett"
+  - "@cvogt"
+api-approvers:
+  - "@jerolimov"
+  - "@cvogt"
+creation-date: 2022-07-27
+last-updated: 2022-08-18
+tracking-link: 
+  - https://issues.redhat.com/browse/ODC-6695
+status: implementable
+---
+
+# Customize Catalogs
+
+## Summary
+
+The OpenShift Console provides a "Developer Catalog" which enables 
+the user to create apps and services based on Operator backed services, helm charts, templates, etc.
+
+Cluster admins can provide additional catalog items by installing operators or adding helm repositories.
+Cluster administrators want to hide these sub-catalogs.
+
+Under the developer catalog,
+there are categories and sub-categories. When users select a sub-category, they can see the list
+of types/sub-catalogs to install. This enhancement enable cluster admins to customize these sub-catalogs.
```

Comment:
We sometimes refer to these as types and sometimes sub-catalogs. I'd recommend consistently referring to them as "types" since it's what's shown in the UI and to avoid confusion between sub-categories and sub-catalogs.

Reply:
Hi @spadgett ,

Updated sub-catalogs to refer as types. PTAL. Thanks.